main.js

```diff
@@ -1,14 +1,35 @@
-let person = {
-  name: "Yoda",
-  designation: "Jedi Master ",
-};
-
-function trainJedi(jediWarrion) {
-  if (jediWarrion.name === "Yoda") {
-    console.log("No need! already trained");
+let user = 'John Doe';
+console.log(user);
+
+let student = 'Eduard';
+console.log(student);
+
+let test = 1;
+test = test + 1;
+test += '1'; // 21;
+console.log(test);
+test -= 1; // 20;
+console.log(test); // true
+console.log(Boolean(test));
+
+let result = 1;
+let arr = [2, 3, 5, 8];
+for (let i = 0; i < 4; i++) {
+  result *= arr[i];
+}
+console.log(result);
+
+let arrOdd = [2, 5, 8, 15, 0, 6, 20, 3];
+for (let i = 0; i < arrOdd.length; i++) {
+  if (arrOdd[i] > 5 && arrOdd[i] < 10) {
+    console.log(arrOdd[i]);
+  }
+}
+
+let arrEven = [2, 5, 8, 15, 0, 6, 20, 3];
+for (let i = 0; i < arrEven.length; i++) {
```

Comment:
fun fact =)
https://ru.wikipedia.org/wiki/%D0%A7%D1%91%D1%82%D0%BD%D0%BE%D1%81%D1%82%D1%8C_%D0%BD%D1%83%D0%BB%D1%8F#:~:text=%D0%9D%D0%BE%D0%BB%D1%8C%20%E2%80%94%20%D1%87%D1%91%D1%82%D0%BD%D0%BE%D0%B5%20%D1%87%D0%B8%D1%81%D0%BB%D0%BE.